app/features/ledger/SignAddIdentityProvider.ts

```diff
@@ -20,6 +21,36 @@ import {
 
 const INS_ADD_IDENTITY_PROVIDER = 0x2d;
 
+/**
+ * Sends the given description for signing.
+ * Uses the given p1 for length and p1 + 1 for the content.
```

Comment:
This could be misunderstood as the `p1` containing some length. Adjust the documentation to make it more clear what is happening.